pom.xml

```diff
@@ -83,6 +82,7 @@
     <dockerfile.maven.extension.version>1.4.9</dockerfile.maven.extension.version>
     <dockerfile.useMavenSettingsForAuth>true</dockerfile.useMavenSettingsForAuth>
     <dockerfile.googleContainerRegistryEnabled>false</dockerfile.googleContainerRegistryEnabled>
+    <reactor.bom.version>Californium-SR3</reactor.bom.version>
```

Comment:
why not Californium-RELEASE?

Reply:
I believe we should take the latest stable release, then whenever we have a need for an update - we update this to the latest SR.

For those who want to catch up:
https://github.com/reactor/reactor#bom-versioning-scheme

> The first stable release is simply suffixed with `-RELEASE`, but the equivalent of patch releases are also possible as "Service Releases", appending the suffix `-SR` followed by the number of the service release (eg. `-SR1`, `-SR2`).

Reply:
let's say in `services` we are using version 3.2.4 of something which is included in SR1, and in cluster we have version 3.2.2 of something else which is included in SR2, the RELEASE version will create both regression in versions and hopefully it will be reflected in compilation errors (and not in runtime!!)